aiven/client/argx.py

```diff
@@ -33,6 +33,22 @@ def default(self, o):  # pylint:disable=E0202
         return jsonlib.JSONEncoder.default(self, o)
 
 
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawDescriptionHelpFormatter):
+    """Help formatter to display the default value only for integers and non-empty strings"""
+    def _get_help_string(self, action):
```

Comment:
Why is `ArgumentDefaultsHelpFormatter` inherited? It only defines the `_get_help_string` method, which this class overrides. Both superclasses inherit from `HelpFormatter`.

Reply:
Dropped ArgumentDefaultsHelpFormatter